x/msgfees/client/cli/tx.go

```diff
@@ -134,3 +137,42 @@ $ %[1]s tx msgfees remove "removing" "removing MsgWriterRecordRequest fee" 10nha
 	cmd.Flags().String(FlagMinFee, "", "additional fee for msg based fee")
 	return cmd
 }
+
+func GetUpdateUsdConversionRateProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "usd-conversion-proposal <title> <description> <rate> <deposit>",
+		Aliases: []string{"ucp", "u-c-p"},
+		Args:    cobra.ExactArgs(4),
+		Short:   "Submit a usd conversion rate update proposal along with an initial deposit",
+		Long: strings.TrimSpace(`Submit a usd conversion rate update proposal along with an initial deposit.
+		The conversion rate is the cost in thousandths of USD (1/10 of a cent) for 1 unit of hash (1,000,000,000nhash).  Example: For example, $1.098/Hash, is a rate of 1098, and $0.035/Hash is a rate of 35
+`),
```

Comment:
Again with the tabs inside ticks.
As it is, this is what the output would look like:
```plaintext
Submit a usd conversion rate update proposal along with an initial deposit.
		The conversion rate is the cost in thousandths of USD (1/10 of a cent) for 1 unit of hash (1,000,000,000nhash).  Example: For example, $1.098/Hash, is a rate of 1098, and $0.035/Hash is a rate of 35
```
```suggestion
		Long: strings.TrimSpace(`Submit a usd conversion rate update proposal along with an initial deposit.
The conversion rate is the cost in thousandths of USD (1/10 of a cent) for 1 unit of hash (1,000,000,000nhash). 
For example, $1.098/Hash, is a rate of 1098, and $0.035/Hash is a rate of 35
`),
```

Reply:
Weir, thought I fixed these yesterday.